style.css

```diff
@@ -1 +1,149 @@
 /*  Add your styling here */
+*{
+    margin: 0%;
+    padding: 0%;
+}
+@import url('https://fonts.googleapis.com/css2?family=Frank+Ruhl+Libre:wght@300&family=Nanum+Gothic&family=Sarabun:wght@100&display=swap');
+
+body{
+    /* width: 390px;
+    height: 844px; */
+    
+}
+.box{
+    padding: 29px;
+    background-color: aqua;
+}
+.contener{
+display: grid;
+grid-template-columns: repeat(2,1fr);
+grid-template-rows: repeat(7,1fr) ;
+grid-gap: 3px;
+grid-template-areas: 
+"hr hr"
+"ar ar"
+"ar ar"
+"b_img b_img"
+"img1 img2"
+"img3 img4"
+"foot foot";
+}
+h1{
+    font-size: 100%;
+    font-family: 'Sarabun', sans-serif;
+
+}
+/* header */
+.header{
+grid-area: hr;
+display: flex;
+/* margin: 0%;
+padding: 0%; */
+margin: 0px;
+padding: 0px;
+justify-content:space-between;
+width: 390px;
+height: 118px;
+background-color: #d4a373;
+}
+.icon1{
+ display: flex;
```

Comment:
Since we are trying to practice grid can you just use grid for this project and leave flex out if you can, if you don't have time to do it is fine but yeah just it is good for you to practice more grid that's all